src/appkit/animation.rs

```diff
@@ -39,7 +39,7 @@ impl AnimationContext {
 
         unsafe {
             //let context: id = msg_send![class!(NSAnimationContext), currentContext];
-            let _: () = msg_send![class!(NSAnimationContext), runAnimationGroup:block];
+            let _: () = msg_send![class!(NSAnimationContext), runAnimationGroup: block];
```

Comment:
Hmmm, the space between `:` and `block` - any reason for this? I try to think of objc message passing here the same way I'd write Objective-C, feels disjointed to read.

(If there's a compelling reason for it then it can happen tho)

Reply:
This is something `rustfmt` did by itself to match the formatting of the rest of the language.

Related, I've changed the `msg_send!` family of macros to support commas between arguments as well, with the plan to deprecate and eventually disallow omitting them. I've done this since the way the macro works currently is not actually valid [according to the reference](https://doc.rust-lang.org/reference/macros-by-example.html#follow-set-ambiguity-restrictions); in particular, an `expr` cannot come after an `ident`, see [this playground link for details](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=4b46d5fea92dfe69fafc0a3f21fd4c31).

In light of the above, I think it's a bit of a wasted endeavour to try to make it look like we're writing Objective-C, so instead I opted to make the syntax more Rust-like - hence I think the formatting that `rustfmt` does here is a good idea, since again, it makes the syntax look more Rust-like. But that's mostly an opinion, by far not a dealbreaker.

Reply:
Ha, reading your reasoning of:

>Also, the previous syntax wasn't even that consistent with Objective-C, we already had to put a comma after the receiver!

Sells me already. I'll all for it in this case. :)